app/helpers/account_activations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
+end
```

Comment:
It seems like this file doesn't help you. I suggest you to remove those empty files.